scripts/get-orig-tarball

```diff
@@ -3,21 +3,28 @@
 VERBOSITY=0
 TEMP_D=""
 FORMATS=".tar.gz .tar.xz .tar.bz2"
-DSC_BASE_URL="http://launchpad.net/ubuntu/+archive/primary/+files"
+DSC_UBUNTU_BASE_URL="http://launchpad.net/ubuntu/+archive/primary/+files"
+DSC_DEBIAN_BASE_URL="http://launchpad.net/debian/+archive/primary/+files"
 
 error() { echo "$@" 1>&2; }
 fail() { [ $# -eq 0 ] || error "$@"; exit 1; }
 
 Usage() {
     cat <<EOF
-Usage: ${0##*/} [ options ] [changelog [dir]]
+Usage: ${0##*/} [ options ] [changelog [dir]] | [subcommand]
 
-   changelog: the changelog to parse.
+   changelog: the changelog to parse. Default: debian/changelog.
    dir: place to put orig tarball. 
         default: dirname(changelog)/../../
 
    options:
       -O | --overwrite: overwrite an existing file 
```

Comment:
I know it's unrelated, but this help test doesn't mention `--offset`, `--unreleased`, `--verbose`.

Reply:
i'd say 

> The new logic LGTM, thanks!
> 
> On my wishlist is a `--orig-only` flag to be used with `pull-pkg --pull=source`, I may prepare a PR adding it at some point :)

i dont' understand.  all get-orig-tarball ever produces is the orig file. (it throws away the other htings that dget downloads).

Reply:
Sorry if I wasn't clear, I wasn't suggesting a change to this script, but to `pull-pkg`, adding a flag to make it throw away the other files only keeping the .orig, like `get-orig-tarball` does. Thus we'd have a way to only download the .orig in a standard dev tool.